Code/GraphMol/Conformer.h

```diff
@@ -104,7 +105,9 @@ class RDKIT_GRAPHMOL_EXPORT Conformer : public RDProps {
 
   //! Set the position of the specified atom
   inline void setAtomPos(unsigned int atomId, const RDGeom::Point3D &position) {
-    // RANGE_CHECK(0,atomId,d_positions.size()-1);
+    if (atomId == std::numeric_limits<unsigned int>::max()) {
+      throw ValueErrorException("atom index overflow");
+    }
     if (atomId >= d_positions.size()) {
```

Comment:
Couldn't you solve this by upcasting to int64 when computing the size?  unsigned int::max would still be addressable.

Reply:
This won't work on 32 bit builds, but do we care?

Reply:
I expect this unsigned int issue might exist in a few places, should we look for them?

Reply:
> This won't work on 32 bit builds, but do we care?

nope. There's loads of code that'll break on 32bit builds

Reply:
> I expect this unsigned int issue might exist in a few places, should we look for them?

I don't see any urgency to it. I'm happy to fix these fuzzing-discovered edge cases as they come up